OpenSim/Wrapping/Python/tests/test_swig_additional_interface.py

```diff
@@ -63,10 +63,9 @@ def test_markAdopted2():
     constr.setConstantDistance(1)
     a.addConstraint(constr)
 
-    # Force requires body names. If not provided, you get a segfault.
-    f = osim.BushingForce()
-    f.setBody1ByName("ground")
-    f.setBody2ByName("body")
+    # Force requires frames. If not provided, you get a segfault.
+    f = osim.BushingForce("ground", "body", osim.Vec3(2, 2, 2), osim.Vec3(1, 1, 1), osim.Vec3(0, 0, 0), osim.Vec3(0, 0, 0))
+
```

Comment:
Hmm..perhaps this test should be revisited. I wanted to test in particular using the default constructor. I wonder if with your changes the segfault goes away.